client/client.go

```diff
@@ -538,3 +542,32 @@ func (c *Client) Collect(ch chan<- prometheus.Metric) {
 var (
 	_ prometheus.Collector = (*Client)(nil)
 )
+
+// argListFromMySqlParams creates an array of strings from the pointer to the parameters for pt-mysql-sumamry
+func argListFromMySqlParams(pParams *agentpb.StartActionRequest_PTMySQLSummaryParams) []string {
+	var args []string
+
+	// Only adds the arguments are valid
+	// If socket valid, socket will be used. Otherwise it will try user and password.
```

Comment:
Ok, but why? Socket can't be used with address (you either use TCP connection with address, or UNIX socket connection), but I think it can be used with username and password.

/cc @percona-csalguero